<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WithdrawInputWidget</class>
 <widget class="QWidget" name="WithdrawInputWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>770</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>50</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>提至地址</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_symbol">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>50</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>LNK</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>415</x>
     <y>50</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>数量</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_address">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>92</y>
     <width>288</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="placeholderText">
    <string>请输入提现地址</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_ammount">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>92</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="placeholderText">
    <string>可提现数量:0</string>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_all">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>100</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>全部提现</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>120</y>
     <width>291</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>120</y>
     <width>291</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_confirm">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>190</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
